src/Bicep.Cli.IntegrationTests/BuildCommandTests.cs

```diff
@@ -183,55 +183,6 @@ public async Task Build_Valid_SingleFile_After_Restore_Should_Succeed(DataSet da
                 actualLocation: compiledFilePath);
         }
 
-        [DataTestMethod]
-        [BaselineData_Bicepparam.TestData(Filter = BaselineData_Bicepparam.TestDataFilterType.ValidOnly)]
-        [TestCategory(BaselineHelper.BaselineTestCategory)]
-        public async Task Build_Valid_Params_File_Should_Succeed(BaselineData_Bicepparam baselineData)
```

Comment:
Will `bicep build <name>.bicepparam` no longer be supported?

Reply:
Yes, because there is no need for it with the newer command

Reply:
I would leave these tests for now since this PR isn't removing .bicepparam support from the build command.

Reply:
Actually that is the case (I didn't have in the pr description originally) but let me know if you think I should do a separate pr for it

Reply:
It's OK to do that in this PR, but I don't see any changes to `BuildCommand.cs`. Did it maybe get left out of the commit?

Reply:
No, they are in todo items (currently being worked on)

Reply:
In that case, I would leave the `BuildCommandTests` as is for now. There's always a slight risk of regression (admittedly, it's a small and mostly abstract risk in this case), so the best practice is to remove tests validating a behavior in the same PR as when that behavior is removed.

Reply:
Just saw your comment. I have removed both the test and logic